fastapi/_compat.py

```diff
@@ -93,7 +93,10 @@ class ModelField:
 
         @property
         def alias(self) -> str:
-            a = self.field_info.alias
+            a = (
+                isinstance(self.field_info.validation_alias, str)
+                and self.field_info.validation_alias
+            ) or self.field_info.alias
```

Comment:
```suggestion
        va = self.field_info.validation_alias
        if isinstance(va, str) and va:
            return va
        a = self.field_info.alias
```
I think this way it's easier to read